src/App/Zinc.DeveloperCenter.Client.Web/projects/zn-developercenter/src/app/screens/adr-list/components/repo-summary/repo-list.component.ts

```diff
@@ -16,6 +16,9 @@ export class RepoListComponent implements OnInit, OnDestroy {
     public repos!: Page<RepositoryListComponent>;
     public fetchedRepos = false;
 
+    public sortedOn: string = 'number';
```

Comment:
Codacy has a fix for the issue: [Type string trivially inferred from a string literal, remove type annotation.](https://app.codacy.com/gh/GSFSGroup/Zinc.DeveloperCenter/pullRequest?prid=9805944)
```suggestion
    public sortedOn = 'number';
```